test/sql/storage/storage_versions.test

```diff
@@ -24,4 +24,4 @@ query I
 SELECT tags FROM duckdb_databases() WHERE database_name LIKE 'empty%' ORDER BY database_name;
 ----
 {storage_version=v1.0.0 - v1.1.3}
-{storage_version=v1.2.0 - v1.2.1}
+{storage_version=v1.2.0 - v1.2.2}
```

Comment:
Isn't this wrong now?
They should just say `v1.0.0+` and `v1.2.0+` ?

Reply:
Thanks! Fixed up.